services/server/src/server/apiv1/verification/solc-json/stateless/solc-json.stateless.handlers.ts

```diff
@@ -34,40 +36,38 @@ export async function verifySolcJsonEndpoint(req: Request, res: Response) {
   const contractName = req.body?.contractName;
   const chain = req.body?.chain;
   const address = req.body?.address;
+  const creatorTxHash = req.body?.creatorTxHash;
 
-  // Get metadata and sources from the Solidity JSON input
-  const metadataAndSourcesPathBuffers =
-    await services.verification.getAllMetadataAndSourcesFromSolcJson(
-      solc,
-      solcJson,
-      compilerVersion,
-    );
-
-  // Create metadata contracts from the files
-  const metadataContracts = await createMetadataContractsFromFiles(
-    metadataAndSourcesPathBuffers,
-  );
+  logger.debug("Request to /verify/solc-json", {
+    chainId: chain,
+    address: address,
+    contractName: contractName,
+    compilerVersion: compilerVersion,
+    creatorTxHash: creatorTxHash,
+    solcJson: solcJson,
```

Comment:
Just a note, I find it not so useful to print large inputs like the solcJson even in the debug mode. It makes harder to scroll the terminal and make sense of what's going on. If needed in debug mode we can always use breakpoints with the same input.

Reply:
By this log, I actually wanted to find the exact json input that caused the server downtimes. Turns out, it was so long that it was truncated by GCP in the end. So indeed not very helpful :/

We can remove the json input again from the log.